mongodb/mongodb.rb

```diff
@@ -404,46 +555,46 @@ def monitor_service(service, metric_group)
   raise e
 end
 
-
 @services.each do |service|
```

Comment:
No where we are defining services to have mongodb_admin as an entry. How will this loop run for both mongodb and mongodb_admin ?